.github/workflows/crowdin-download.docs.yml

```diff
@@ -235,6 +235,41 @@ jobs:
           # Uncomment below to debug
           # dryrun_action: true
 
+        env:
+          GITHUB_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_PAT }}
+          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID_DOCS }}
+          CROWDIN_PERSONAL_TOKEN:
+            ${{ secrets.CROWDIN_CAMPERBOT_SERVICE_TOKEN }}
```

Comment:
Silly Question

Do we really need to constantly redeclare the `env` variables for each step?

Reply:
y'know...
No, we don't. We could declare them top level instead.